cmd/scollector/collectors/redis_unix.go

```diff
@@ -286,7 +286,10 @@ func redisInit() {
 			}
 			port := strings.Split(sp[2], ":")[1]
 			if port != "0" {
-				add(port)
+				_, err := util.Command(time.Second, nil, "grep", "-q", "cpu:/docker", fmt.Sprintf("/proc/%s/cgroup", sp[0]))
```

Comment:
Would definitely prefer that this compare the /proc/%s/ns/proc link to pid 1's proc NS, rather than checking for the 'docker' string in the cgroups. Docker isn't the only container system folks are likely to use.

If that is truly difficult, then I'm fine with leaving this as-is for the time being, but it needs a FIXME comment.